timelock-impl/src/test/java/com/palantir/atlasdb/timelock/lock/watch/ArrayLockEventSlidingWindowTest.java

```diff
@@ -101,24 +182,11 @@ private void assertContainsNextEventsInOrder(long version, int startInclusive, i
                         .collect(Collectors.toList()));
     }
 
-    private static LockWatchEvent createEvent(long sequence) {
-        return ImmutableFakeLockWatchEvent.of(sequence, 1);
+    private void addLockEventWithMetadata(Set<LockDescriptor> lockDescriptors, Optional<LockRequestMetadata> metadata) {
+        slidingWindow.add(LockEvent.builder(lockDescriptors, TOKEN, metadata));
     }
 
-    @Value.Immutable
-    abstract static class FakeLockWatchEvent implements LockWatchEvent {
-        @Override
-        @Value.Parameter
-        public abstract long sequence();
-
-        @Override
-        @Value.Parameter
-        public abstract int size();
-
-        @Override
-        public <T> T accept(LockWatchEvent.Visitor<T> visitor) {
-            // do nothing
-            return null;
-        }
```

Comment:
Had to remove this because the `accept` returning null here will cause the `add` to fail since an `Optional<LockRequestMetadata>` is now null Also, we should never return null anyways, no even in test code!.